src/site/markdown/index.md.vm

```diff
@@ -24,10 +24,11 @@ Description
   This version of the plugin uses
   `accumulo-minicluster-$context.get("accumulo.version")`. This version [can
   be overridden similar to overriding checkstyle versions][2] by a later 2.x
-  version. Because Accumulo follows [Semantic Versioning][3], and this plugin
-  limits itself to using only public API, this should work if you meet the
-  dependency requirements of that version (accumulo-minicluster-2.1.0 will
-  require Java 11, for example).
+  version. For an example of setting Accumulo, Hadoop, and Zookeeper versions
```

Comment:
I think the new sentence you added reads better as a new paragraph which follows this one, instead of being inserted into this one.

Reply:
I was thinking it was more information about dependencies like the previous sentence.

Reply:
The way I read it, there is *fact of being able to override*, followed immediately by *version constraints for override*, and then an example at the end. With the sentence in the middle, it seems to interrupt the rules about overriding, before those rules are finished being laid out.

But, either way works well enough. I don't care that much.